clients/python-wrapper/lakefs/server_config.py

```diff
@@ -1,14 +1,15 @@
 from typing import NamedTuple, Optional
 
 import lakefs_sdk
+from pydantic import StrictBool
 
 from lakefs.client import Client, DefaultClient
 
 
 class ServerStorageConfiguration(NamedTuple):
     blockstore_type: str
-    pre_sign_support: bool
-    import_support: bool
+    pre_sign_support: StrictBool
```

Comment:
Since we include Pydantic anyway with `lakefs_sdk` I'm all for this!
However.. if we do use it, I'd suggest `ServerStorageConfiguration` inherits from `pydantic.BaseModel` instead

Reply:
I'll revert it since we use the NamedTuple to easily initialize the class using the SDK object's properties dict